packages/koa-shopify-graphql-proxy/src/shopify-graphql-proxy.ts

```diff
@@ -8,6 +8,7 @@ export enum ApiVersion {
   April19 = '2019-04',
   July19 = '2019-07',
   Unstable = 'unstable',
+  Unversioned = 'unversioned',
```

Comment:
For my understanding, what is the difference between `Unstable` and `Unversioned` ?

Is it worth including comments to provide some context into what each means?

Reply:
`Unversioned` is more stable than `Unstable` and will be only available to 1p app.